appwidget-host/src/main/java/com/google/android/glance/appwidget/host/glance/GlanceRemoteViewsKtx.kt

```diff
@@ -82,7 +83,7 @@ private suspend fun GlanceAppWidget.compose(
     context = context,
     size = size,
     state = state,
-    content = { Content() }
+    content = { Text(text = "Content() missing") }  // TODO Content() composable is not available anymore
```

Comment:
Unfortunately it is not possible to get the a `@Composable` from a `GlanceAppWidget` anymore. Any idea how to work around this?

Reply:
Its like this now:
```kotlin
override suspend fun provideGlance(context: Context, id: GlanceId) {
        provideContent {
            SampleGlanceWidgetContent()
        }
    }
```

Reply:
Maybe I'm missing something, but I still don't get how can `GlanceRemoteViewsKtx` provide a content composable, since `SampleGlanceWidgetContent()` is defined outside this library...

Reply:
Indeed, I was thinking about calling provideGlance and somehow get content out of it, but it doesn't seem easy if possible at all. [scratching head].

Reply:
Another option would be to subclass GlanceAppWidget and provide composable content, but that would require runtime dependency on a new module.